### PR TITLE
add post new-conan-release-1-25

### PR DESCRIPTION
**Pull request by solvingj**
In preparation of 1.25 release, subject to change until release is out.

### CODE REVIEW

**Comment by uilianries** [comment]
I would add Components to this blog post: https://github.com/conan-io/conan/pull/6871

Now we can customize the target, which is very cool for Find CMake generator (e.g. OpenSSL:Crypto OpenSSL:SSL).

File: _posts/2020-05-04-New-conan-release-1-25.markdown [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+settings during the builds of recipes in the `host` context. Otherwise it will
+represent the `build` context. Also, it's not uncommon for cross-building 
+cases when you're in one context and need to access the `settings` from the 
+other context. For these cases, `settings_build` and `settings_target` enable 
+access to both contexts under all circumstances.
+
+## Additional Package Component Modeling
+
+In this release, we make another large stride in the journey to provide
+robust modeling of "components" within packages.  The modeling is delcared
```
Comment (memsharded) [inline _posts/2020-05-04-New-conan-release-1-25.markdown:39]:
```suggestion
robust modeling of "components" within packages.  The modeling is declared
```

File: _posts/2020-05-04-New-conan-release-1-25.markdown [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+## Resumable Downloads 
+
+Downloading files is a frequent activity for the Conan client. For a 
+variety of reasons, downloads can fail. This can be frustrating for any user,   
+but for those users working with very large binaries, repeated failures
+of downloads of such binaries can become a blocker very easily.  This impact
+can be felt within local developer workflows as well as in CI builds.  
+
+In this release, Conan learned the ability to resume downloads which have 
+failed. Because Conan already had [logic to retry failed transfers]
```
Comment (memsharded) [inline _posts/2020-05-04-New-conan-release-1-25.markdown:128]: Maybe clarify that the previous retries were doing a new download, not resuming the broken one?

File: _posts/2020-05-04-New-conan-release-1-25.markdown [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+
+## New HTML Table
+
+Finally, Conan has received a cosmetic upgrade in the form of a new HTML table
+output for the Conan Search command. This is part of an ongoing effort to 
+improve overall UX surrounding searching with Conan. Here is a preview of the 
+new table: 
+
+<p class="centered">
+    <img src="{{ site.url }}/assets/post_images/2020-05-04/new-conan-search-table.png" align="center" alt="New Conan Search HTML Table Output"/>
```
Comment (memsharded) [inline _posts/2020-05-04-New-conan-release-1-25.markdown:141]: Adjust the date for tomorrow, we will be publishing this tomorro 7-May

File: _posts/2020-05-04-New-conan-release-1-25.markdown [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+layout: post 
+comments: false 
+title: "Conan 1.25: New Cross-Build Variables, Additional Package Component Modeling, Automatic Config Install, Resumable Downloads, New HTML Table"
+---
+
+Conan 1.25 follows up the previous release with another wave of progress related
+to the new cross-build modeling. There are also some brand new features we are
+excited about. Conan has learned to update it's configurations and profiles 
+automatically by enabling the ability to schedule the 
+`conan config install command`. We've also dramatically improved the output of
```
Comment (jgsogo) [inline _posts/2020-05-04-New-conan-release-1-25.markdown:11]:
```suggestion
`conan config install` command. We've also dramatically improved the output of
```

File: _posts/2020-05-04-New-conan-release-1-25.markdown [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+## Cross-Build Modeling + Context Modeling  
+
+In the previous release, we introduced the new abstraction of ["contexts"](https://docs.conan.io/en/latest/devtools/build_requires.html#build-and-host-contexts)
+along with some core functionality surrounding them. However, the implementation 
+was still not complete enough for recipes which need this functionality
+to be refactored to use the new model.  In this release, we fill the gaps so 
+that recipes can now start to take advantage of the new model.  
+1.25 features two additional variables on the conanfile class which can be
+accessed and evaluated by recipes:
+- `settings_build`
```
[the remark this answers is not in the document]
Comment (jgsogo) [reply _posts/2020-05-04-New-conan-release-1-25.markdown:24]: Something like this?

image

[the remark this answers is not in the document]
Comment (memsharded) [reply _posts/2020-05-04-New-conan-release-1-25.markdown:24]: Yes, perfect

File: _posts/2020-05-04-New-conan-release-1-25.markdown (hunk 2 above)
Comment (solvingj) [reply _posts/2020-05-04-New-conan-release-1-25.markdown]: Done

File: _posts/2020-05-04-New-conan-release-1-25.markdown (hunk 3 above)
Comment (solvingj) [reply _posts/2020-05-04-New-conan-release-1-25.markdown]: Done

**Pull request merged**